src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1324,6 +1360,10 @@ void goto_instrument_parse_optionst::help()
     "Semantic transformations:\n"
     " --nondet-volatile            makes reads from volatile variables non-deterministic\n"
     " --unwind <n>                 unwinds the loops <n> times\n"
+    " --unwindset L:B,...          unwind loop L with a bound of B\n"
+    " --partial-loops              permit paths with partial loops\n"
+    " --unwinding-assertions       generate unwinding assertions\n"
+    " --rest-loops                 add loop for remaining iterations after unwound part\n"
```

Comment:
I find "rest-loops" hard to parse/understand. "remaining" instead of "rest" would seem clearer to me, but there might be other ideas?